_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -645,6 +667,21 @@ public static function get_site_data() {
 		return new WP_Error( 'site_id_missing', esc_html__( 'The ID of this site does not exist.', 'jetpack' ), array( 'status' => 404 ) );
 	}
 
+	/**
+	 * Sets a flag
+	 *
+	 * @since 4.4.0
+	 *
+	 * @return bool True
+	 */
+	public static function confirm_safe_mode() {
+		Jetpack_Options::update_option( 'safe_mode_confirmed', true );
```

Comment:
Since `Jetpack_Options::update_option()` returns a bool, should we maybe do something like:

```php
$updated = Jetpack_Options::update_option( 'safe_mode_confirmed', true );
return rest_ensure_response(
    array(
      'code' => $updated ? 'success': 'error'
    )
);
```